websites/D/Discords.com/presence.ts

```diff
@@ -3,351 +3,508 @@ const presence = new Presence({
 	}),
 	browsingTimestamp = Math.floor(Date.now() / 1000);
 
+enum Assets {
+	Logo = "https://i.imgur.com/ZjtyAQx.png",
+	Bot = "https://i.imgur.com/Z6ruEIy.png",
+	Heart = "https://i.imgur.com/oKCHv5x.png",
+}
+
 presence.on("UpdateData", async () => {
-	const showTimestamp = await presence.getSetting<boolean>("timestamp"),
-		showButtons = await presence.getSetting<boolean>("buttons"),
-		presenceData: PresenceData = {
-			largeImageKey: "discords_logo",
-		};
+	let presenceData: PresenceData = {
+		largeImageKey: Assets.Logo,
+	};
+
+	const { href, pathname } = document.location,
+		[showTimestamp, showButtons] = await Promise.all([
+			presence.getSetting<boolean>("timestamp"),
+			presence.getSetting<boolean>("buttons"),
+		]),
+		pages: Record<string, PresenceData> = {
+			"/u/dashboard": {
+				details: "⚙️ Viewing Dashboard",
+				buttons: [{ label: "View Page", url: href }],
+			},
+			"/premium": {
+				details: "💎 Viewing Premium Plans",
+				buttons: [{ label: "View Page", url: href }],
+			},
+			"/emoji-list": {
+				details: "😃 Viewing Emoji list",
+				buttons: [{ label: "View Emojis", url: href }],
+			},
+			"/sticker-list": {
+				details: "🖼️ Viewing Sticker list",
+				buttons: [{ label: "View Stickers", url: href }],
+			},
+			"/bots": {
+				details: "🤖 Viewing Bots",
+				buttons: [{ label: "View Bots", url: href }],
+			},
+			"/templates": {
+				details: "📁 Viewing Templates",
+				buttons: [{ label: "View Templates", url: href }],
+			},
+			"/termsofservice": {
+				details: "📜 Viewing Terms of Service",
+				buttons: [{ label: "View Page", url: href }],
+			},
+			"/verification": {
+				details: "🔐 Viewing Verification",
+				buttons: [{ label: "View Page", url: href }],
+			},
+			"/apply": {
+				details: "📝 Applying for Verification",
+				buttons: [{ label: "Apply", url: href }],
+			},
+		},
```

Comment:
Previously would have been allowed, but nowadays we don't really approve the use of emojis

Reply:
Will change.